easybuild/easyconfigs/j/Julia/Julia-1.5.1-foss-2020a.eb

```diff
@@ -0,0 +1,79 @@
+easyblock = 'ConfigureMake'
+
+name = 'Julia'
+version = '1.5.1'
+
+homepage = 'https://julialang.org'
+description = "Julia is a high-level, high-performance dynamic language for technical computing."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'rpath': True}
+
+source_urls = ['https://github.com/JuliaLang/julia/releases/download/v%(version)s/']
+sources = ['julia-%(version)s-full.tar.gz']
+checksums = ['1d0debfccfc7cd07047aa862dd2b1a96f7438932da1f5feff6c1033a63f9b1d4']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('CMake', '3.16.4'),
+    ('Python', '3.8.2'),
+    ('Perl', '5.30.2'),
+    ('pkg-config', '0.29.2'),
+    ('PCRE2', '10.34'),
```

Comment:
Is PCRE2 really a build dep? I suppose, maybe it statically links? But we use PCRE2 as a runtime dep in R for example.

Reply:
I saw it listed in the `Makefile` as a build dep.